pkg/e2e/operators/certman.go

```diff
@@ -15,7 +16,22 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
-var _ = ginkgo.Describe("[Suite: operators] [OSD] Certman Operator", func() {
+var testAlert alert.MetricAlert
+
+func init() {
+	ma := alert.GetMetricAlerts()
+	testAlert = alert.MetricAlert{
+		Name:             "[Suite: operators] [OSD] Certman Operator",
+		TeamOwner:        "SD-SREP",
+		PrimaryContact:   "Carl Brumm",
```

Comment:
i hate to stick our collective head in the lion's mouth, but... @cblecker should this be our team?

Reply:
Yeahhhhh. Put me here :(

Reply:
Contain your excitement. ;p Also. updated.